docs/content/en/docs/reference/tetragon-configuration.md

```diff
@@ -12,65 +12,7 @@ as a CLI arguments or as configuration options from [YAML](https://yaml.org) fil
-      --export-file-rotation-interval duration    Interval at which to rotate JSON export files in addition to rotating them by size
-      --export-filename string                    Filename for JSON export. Disabled by default
-      --export-rate-limit int                     Rate limit (per minute) for event export. Set to -1 to disable (default -1)
-      --expose-kernel-addresses                   Expose real kernel addresses in events stack traces
-      --field-filters string                      Field filters for event exports
-      --force-large-progs                         Force loading large programs, even in kernels with < 5.3 versions
-      --force-small-progs                         Force loading small programs, even in kernels with >= 5.3 versions
-      --gops-address string                       gops server address (e.g. 'localhost:8118'). Disabled by default
-  -h, --help                                      help for tetragon
-      --k8s-kubeconfig-path string                Absolute path of the kubernetes kubeconfig file
-      --kernel string                             Kernel version
-      --kmods strings                             List of kernel modules to load symbols from
-      --log-format string                         Set log format (default "text")
-      --log-level string                          Set log level (default "info")
-      --metrics-server string                     Metrics server address (e.g. ':2112'). Disabled by default
-      --netns-dir string                          Network namespace dir (default "/var/run/docker/netns/")
-      --process-cache-size int                    Size of the process cache (default 65536)
-      --procfs string                             Location of procfs to consume existing PIDs (default "/proc/")
-      --rb-queue-size int                         Set size of channel between ring buffer and sensor go routines (default 65k) (default 65535)
-      --rb-size int                               Set perf ring buffer size for single cpu (default 65k)
-      --rb-size-total int                         Set perf ring buffer size in total for all cpus (default 65k per cpu)
-      --release-pinned-bpf                        Release all pinned BPF programs and maps in Tetragon BPF directory. Enabled by default. Set to false to disable (default true)
-      --server-address string                     gRPC server address (e.g. 'localhost:54321' or 'unix:///var/run/tetragon/tetragon.sock' (default "localhost:54321")
-      --tracing-policy string                     Tracing policy file to load at startup
-      --tracing-policy-dir string                 Directory from where to load Tracing Policies (default "/etc/tetragon/tetragon.tp.d")
-      --verbose int                               set verbosity level for eBPF verifier dumps. Pass 0 for silent, 1 for truncated logs, 2 for a full dump
-```
+{{< readfile "/docs/reference/daemon-flags.md" >}}
```

Comment:
oh nevermind!!

Reply:
I think it is fine right? I mean who will go modify the daemon-flags.md without checking where it is used?

Reply:
yes it's good sorry I thought you didn't `readfile` like that! :)